cmd/apiKeyValueMapUpload_test.go

```diff
@@ -48,9 +49,8 @@ func TestRunApiKeyValueMapUpload(t *testing.T) {
 		// test
 		payload, err := createJSONPayload(&config)
 		// assert
+		require.NoError(t, err)
```

Comment:
Why did this commit not work?

Reply:
require is failing the "Go / unit (pull_request) " test every time, tried 2 times.